### ISSUE TITLE
Display disk usage for each volume

### ISSUE THREAD

**Issue by pdirksen** [resin-io-modules/drivelist#262]
Please add the disk usage (total, used, available) for each volume.

Windows: `wmic logicaldisk get` has attributes `Size`,`FreeSpace`
Linux:  `df -T` has  `Used`,`Available`
MacOS: `diskutil info -all`  should have those as well

**Comment by lurch** [comment]
Duplicate of #228 ?

**Comment by jhermsmeier** [comment]
Yeah, we've dismissed it before, but might be worth having another look at. I don't think those values will be accurate though, as they'll depend on which file systems the respective OS support – in which case I don't think it'd make sense to put in.

**Comment by thom-nic** [comment]
I'd be interested in this as well; in all of the most common cases it's easy to get used/free from the filesystem.  I'm using drivelist on an embedded linux device and one of my motivating use cases is monitoring disk usage.
